pynucastro/networks/python_network.py

```diff
@@ -143,7 +203,23 @@ def screening_string(self, indent=""):
         return ostr
 
     def rates_string(self, indent=""):
-        """section for evaluating the rates and storing them in rate_eval"""
+        """Create the python code that calls the evaluation function for each
+        rate.  Typically this is of the form `name(rate_eval, ...)`, where
+        `rate_eval` is the `PythonNetwork` class that stores the rate values.
```

Comment:
`rate_eval` is an instance of the generated `RateEval` class, right? Also, should these be double backquotes for code, since this is ReST?

Reply:
updated

turns out RateEval is only created when we write out the network, so I adjusted the language